### PR TITLE
Node AutoComplete suggestions sorted alphabetically within the same group

### PR DESCRIPTION
**Pull request by zeusongit**
### Purpose

[DYN-3253](https://jira.autodesk.com/browse/DYN-3253)
Node AutoComplete suggestions will be sorted alphabetically within the same group

Before:
beforenauto

After:
Screen Shot 2020-11-24 at 6 50 29 PM

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

**Comment by QilongTang** [comment]
@zeusongit Excellent, is there a trivial unit test we can write just to guard this order?

File: test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs [hunk 1]
```diff
@@ -255,6 +255,40 @@ public void NodeSuggestions_DefaultSuggestions()
...
+                "DSCoreNodes.DSCore.Color.ByARGB",
+                "DSCoreNodes.DSCore.Color.Divide",
+                "DSCoreNodes.DSCore.Color.Multiply",
+                "DSCoreNodes.DSCore.ColorRange.GetColorAtParameter",
+                "DSCoreNodes.DSCore.IO.Image.Pixels"};
+
+            for (int i = 0; i < 6; i++)
+            {
+                Assert.AreEqual(expectedNodes.ElementAt(i), suggestedNodes.ElementAt(i));
+            }
```
Comment (QilongTang) [inline test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs:272]: Maybe use https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=net-5.0

Comment (QilongTang) [reply test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs:272]: There are some other alternatives in https://stackoverflow.com/questions/37020277/c-sharp-how-to-check-if-list-of-items-are-in-alphabetical-order-or-not.

Comment (zeusongit) [reply test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs:272]: updated

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
@zeusongit Thanks, LGTM. And if all regressions pass after this change, please merge and cherry-pick.

**Pull request merged**